src/components/tutorials-landing-view/types.ts

```diff
@@ -40,6 +40,7 @@ interface CertificationContentCardLinkProps {
 		description: string
 	}
 	product: {
+		name: ProductName
```

Comment:
If you're rightfully wondering how I just added this key to the type without any changes to the actual value, see [here](https://github.com/hashicorp/dev-portal/blob/81de0cff7bc3457cdb3a07bd177f6ca285450bf3/src/views/tutorials-landing/components/product-section/index.tsx#L31).

The type had 2 missing keys from where the prop got passed down initially, I only opted to add this one since it's what's need to discriminate between `Consul` and other products